conftest.py

```diff
@@ -23,6 +23,8 @@ def get_test_devices() -> Dict[str, torch.device]:
         import torch_xla.core.xla_model as xm
 
         devices["tpu"] = xm.xla_device()
+    if torch.backends.mps.is_available():
```

Comment:
```suggestion
    if torch.backends.mps.is_available(): # type: ignore
```
or possible add a small function like `kornia.mps_is_available()` that checks first if the module exists or similar